Client/API/ApiRequestHandler.cs

```diff
@@ -77,6 +77,19 @@ public async Task<IEnumerable<Country>> FetchAllCountries(HttpClient httpClient)
             }
         }
 
+        public async Task<IEnumerable<Country>> FetchAllCountryIdentifiers(HttpClient httpClient)
+        {
+            try
+            {
+                return await httpClient.GetFromJsonAsync<IEnumerable<Country>>($"{apiUrl}/Country/GetAllCountryIdentifiers");
```

Comment:
I would suggest using the same getasync + readasync combo that the other methods use. This makes it possible to test the methods.